tests/extension.test.ts

```diff
@@ -58,12 +58,32 @@ test('read query is executed against replica', async () => {
   expect(logs).toEqual([{ server: 'replica', operation: 'findMany' }])
 })
 
+test('$queryRaw and $queryRawUnsafe are executed against replica', async () => {
+  await expect(prisma.$queryRaw`SELECT ${'asdf'} as id`).resolves.toEqual([{ id: 'asdf' }])
+  await expect(prisma.$queryRawUnsafe('SELECT $1 as id', 'asdf')).resolves.toEqual([{ id: 'asdf' }])
```

Comment:
What would happen with an `INSERT` or `UPDATE` via `$queryRaw`?

Reply:
It would still go to the replica, which I think is obvious from the code but maybe not from the readme. I'll go ahead and add information to the readme as well.

Reply:
But a `INSERT` or `UPDATE` against the read-only replica would fail.
Right now, as raw queries are not marked as read-only queries, this is not the case and you are protected from that.
Does this PR then not introduce a foot gun for the users?